systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	restoreLocation = "/data/backup/"
+	backupDst       = "/data/backup/"
+	headerAlpha1    = http.Header{}
+	headerAlpha2    = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	ShellToUse      = "bash"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backup/")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	// *****************Get JWT-Token
+
+	jwtTokenAlpha1 := testutil.GrootHttpLogin("http://" + testutil.SockAddrHttp + "/admin").AccessJwt
+	headerAlpha1.Set(accessJwtHeader, jwtTokenAlpha1)
+	fmt.Println("Header1: ", jwtTokenAlpha1)
```

Comment:
Were these leftover from when you were writing the tests? In most cases, tests should be silent unless failed.

Reply:
**Resolved:** Removed excessive print statements from the code.